keystore.gradle.txt

```diff
@@ -0,0 +1,4 @@
+storeFile="relative path to your keystore file"
+storePassword="keystore file pass"
+keyAlias=
+keyPassword=
```

Comment:
Remove this file

Reply:
if i romove it the project dont work... show  me error at compile

i use de help from https://github.com/Wolox/WoloxAndroidBootstrap
![image](https://user-images.githubusercontent.com/52514937/126185685-0a64e8ab-077f-4928-8bbb-883615772707.png)

Reply:
This is not the original file. The original file is located in `/app/keystore.gradle`, this file is named `keystore.gradle.txt`